src/shadow-embed.js

```diff
@@ -212,7 +216,14 @@ export function importShadowBody(shadowRoot, body, deep) {
     }
   }
   setStyle(resultBody, 'position', 'relative');
-  shadowRoot.appendChild(resultBody);
+  resultBody.classList.add('amp-shadow');
+  //NOTE(torch2424), create a shadow root elelment, and add it to the body
+  //DO NOT SUBMIT: Find a solution, this is wrong place. Fix the problem here, need to place styles differently
+  //    getRootNode().firstChild
+  const shadowRootElement = doc.createElement('i-amphtml-shadow-root');
```

Comment:
Done. I reworded the comment, please let me know if it sounds correct, opposed to before.